README.md

```diff
@@ -1,100 +1,132 @@
-# Deploy a Testnet
+# Connect to a Testnet
 
-## Install Cosmos-SDK on a Digital Ocean Droplet \(Recommended\)
+This document explains how to connect to the Testnet of a [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) based blockchain. It can be used to connect to the latest Testnet for the Cosmos Hub.
+
+## Software Setup (Manual Installation)
+
+Follow this instructions to install the Cosmos-SDK and connect to the latest Testnet. This instructions work for both a local machine and a VM in a cloud server.
+
+If you intend to run a non-validator full-node, installing the SDK on a Cloud server is a good option. Although, if you are want to become a validator for the `mainnet` you should look at more complex setups to protect your node from DDOS and ensure high-availability (see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). You can check more information regarding validators in our [website](https://cosmos.network/validators) or in the [validator FAQ](https://cosmos.network/resources/validator-faq).
```

Comment:
You can get more information regarding validators in our website, in the Validator FAQ or in the Validator Chat
(add chat)